style.css

```diff
@@ -11,24 +11,45 @@
 
 body {
   /* I prefer sans-serif (without curls) so I'm changing it haha */
-  font-family: Helvetica, sans-serif;
+	/* lets see what Arial looks like*/
+  font-family: Arial;
+background-color: #afeeee;
 }
 
+
+h1 {
+
+color: #680000;
+background-color: #003362
+
```

Comment:
You have removed the closing `}` for the `h1` here and so the rest will be read incorrectly by the browser